packages/suite/src/support/messages.ts

```diff
@@ -1193,7 +1193,7 @@ const definedMessages = defineMessages({
     },
     TR_DEVICE_IS_USED_IN_OTHER_WINDOW_TEXT: {
         defaultMessage:
-            'This is a big no no. Please dont use device in other window. Close all other windows or tabs that might be using your Trezor device.',
+            "This is a big no no. Please don't use device in other window. Close all other windows or tabs that might be using your Trezor device.",
```

Comment:
this is already updated in crowdin, if you are gonna edit the default message use one from crowdin otherwise you are going to invalidate proper copy